docs/agents/sematext-agent/ansible.md

```diff
@@ -0,0 +1,129 @@
+
+```
+server1 | SUCCESS => {
+    "ansible_facts": {
+        "discovered_interpreter_python": "/usr/bin/python3"
+    },
+    "changed": false,
+    "ping": "pong"
+}
+server2 | SUCCESS => {
+    "ansible_facts": {
+        "discovered_interpreter_python": "/usr/bin/python3"
+    },
+    "changed": false,
+    "ping": "pong"
+}
+```
+
+
+## Sematext Agent Ansible Role
+The Sematext Agent role contains everything required to install Sematext Agent in a specified list of hosts according to the Ansible playbook that will be used. It can be installed either through Ansible Galaxy or manually from the [ansible-install-sematext repository](https://github.com/sematext/ansible-install-sematext).
+
+
+### Install from Ansible Galaxy
+Install the [Sematext Agent role from Ansible Galaxy](https://galaxy.ansible.com/ui/standalone/roles/sematext/spm-monitor-install/) on your control node by issuying the following command:
+
+```ansible-galaxy role install sematext.spm-monitor-install```
+
+
+### Manual Installation
+If you want to manually install the role, clone the [ansible-install-sematext](https://github.com/sematext/ansible-install-sematext) repo in the control node and copy this repo to `~/.ansible/roles/sematext.spm-monitor-install`.
+
+
+### Supported Platforms
+Linux distributions:
+
+- Ubuntu
+    - xenial
+    - bionic
+    - focal
+    - jammy
+- Debian
+    - wheezy
+    - jessie
+    - stretch
+    - buster
+    - bullseye
+    - bookworm
+- Red Hat Enterprise Linux 6, 7, 8, 9
+- Fedora 38
+- Amazon Linux 2
+
+## Deploying Sematext Agent
+
+### Create an Infra App in Sematext Cloud
+After you get logged into Sematext Cloud at https://apps.sematext.com (or https://apps.eu.sematext.com if using Sematext Cloud Europe), the first step is to [create an Infra App](https://sematext.com/docs/monitoring/infrastructure/#create-an-infra-app) to start monitor your infrastructure. After entering the App name, select Ansible from the list of the environments. Make sure to copy the provided infra token, it will be needed later to pass it as a var in your Ansible playbook.
+
+### Install the agent using an Ansible playbook
+You can integrate the Sematext Ansible role in your existing playbooks or create a new one. Here is an example playbook:
+
+```
+- hosts: all
+  gather_facts: yes
+  become: yes
+  vars:
+    infra_token: INFRA_TOKEN
+  roles:
+    - { role: sematext.spm-monitor-install }
+
+```
+
+`INFRA_TOKEN` is your infra token that you copied earlier from Sematext Cloud.
+
+Run your playbook using the following command:
+
+```ansible-playbook -i hosts sematext-agent-playbook.yaml```
+
+where `hosts` is your Ansible inventory and `sematext-agent-playbook.yaml` is the name of your playbook.
+
+This command will deploy Sematext agent in all hosts defined in your playbook. As soon as the command completes successfully, you can visit Sematext Cloud and see all your hosts online under Infrastructure.
```

Comment:
defined in your playbook or defined in your inventory?

Reply:
it should say "defined in your inventory".

Reply:
No. All the hosts are indeed defined in the inventory but we use the playbook to select specific host groups. So "defined in your inventory" is only true if I have `hosts: all` in the playbook. But I could use `hosts: webservers` in the playbook.